inc/3rd-party/themes/impreza.php

```diff
@@ -0,0 +1,24 @@
+<?php
+defined( 'ABSPATH' ) || die( 'Cheatin&#8217; uh?' );
+
+$current_theme = wp_get_theme();
+
+if ( 'impreza' === strtolower( $current_theme->get( 'Name' ) ) || 'impreza' === strtolower( $current_theme->get( 'Template' ) ) ) {
+	/**
+	 * Excludes Impreza files from minification/combine, defer and cache busting
+	 *
+	 * @since 3.2
+	 * @author TheZoker
+	 *
+	 * @param array $excluded_js Array of JS filepaths to be excluded.
+	 * @return array
+	 */
+	function rocket_exclude_js_impreza( $excluded_js ) {
```

Comment:
too much indentation here

Reply:
I think it's too less on the `add_filter` lines instead. I fixed it